src/common/Nb/index.tsx

```diff
@@ -0,0 +1,47 @@
+import * as S from './Nb.style';
+import { useState } from 'react';
+import { MAIN_BUTTON_DATA } from '../../constant/MAIN_BUTTON';
+
+export interface NbButtonProps {
+  label: string;
+  index: number;
+  onClick: (index: number) => void;
+  isActive: boolean;
+}
+
+//버튼 안에 라벨 넣기
+export const NbButton = ({ label, index, onClick, isActive }: NbButtonProps) => {
+  return (
+    <S.NbBtn onClick={() => onClick(index)} isActive={isActive}>
+      {label}
+    </S.NbBtn>
+  );
+};
+
+const Nb = () => {
+  const [isClicked, setIsClicked] = useState(0);
+  const [, setIsActive] = useState(false);
+
+  // 화면 내 키 전송을 통한 이동용
+  const moveToPage = (index: number) => {
+    setIsClicked(index);
+    setIsActive(true);
+  };
+
+  // map을 통해서 버튼 생성
+  return (
+    <S.NbWrapper>
+      {MAIN_BUTTON_DATA.map((button, index) => (
+        <NbButton
+          key={index}
+          label={button.label}
```

Comment:
constant > MAIN_BUTTON에 코멘트 남긴 것처럼 코드를 수정하게 된다면, 요기는 아래처럼 바꿔줘야겠죵 !?
```suggestion
          label={button}
```